src/splitInteger.test.js

```diff
@@ -4,18 +4,35 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const result = splitInteger(17, 4);
 
+  expect(result).toEqual([4, 4, 4, 5]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
+  const result = splitInteger(8, 1);
 
+  expect(result).toEqual([8]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
+  const result = splitInteger(32, 6);
+  const sortedResult = [...result].sort((a, b) => a - b);
 
+  expect(result).toEqual(sortedResult);
 });
```

Comment:
This test case seems to be testing the sorting of the array, but the `splitInteger` function should not necessarily return a sorted array. The important thing is that it splits the integer into the specified number of parts, not the order of the parts. Consider changing this test or adding a clarification comment.